### ISSUE TITLE
should `pathways_cbd` be a function?

### ISSUE THREAD

**Issue by PietrH** [trias-project/trias#134]
Currently `pathways_cbd()` is a function that accepts no arguments, and always returns the same tibble. Woudn't it be better to include this tibble as a data object? 

https://usethis.r-lib.org/reference/use_data.html

https://github.com/trias-project/trias/blob/98ef3f2afacdd506ed54d3ba42891c8b71433634/R/pathways_cbd.R#L1-L77

**Comment by damianooldoni** [comment]
You are right. I was in that time following rgbif's maintainer, Scott Chamberlain, which was using something similar. But you are right.

**Comment by PietrH** [comment]
Scott might have a good reason, I'd be interested to know the advantages of doing it this way. 

Can we link this issue to a branch or PR? That way we don't lose track. 

pathways_cbd perhaps?
